File: python/program4.py

start = int(input("Enter the start of the range: "))
end = int(input("Enter the end of the range: "))
print("Even numbers in reverse order:")
for i in range(end, start-1, -1):
    if i % 2 == 0:
        print(i, end=" ")

print("\n")  
print("Odd numbers in standard way:")
for i in range(start, end+1):
    if i % 2 != 0:
        print(i, end=" ")

File: Cprogramming/For -{Pattern Problems}/simple nested for loop 5/program4.c
#include <stdio.h>

int main() {
    int start, end;
    printf("Enter start number: ");
    scanf("%d", &start);
    printf("Enter end number: ");
    scanf("%d", &end);

    printf("Even numbers in reverse order: ");
    for (int i = end; i >= start; i--) {
        if (i % 2 == 0)
            printf("%d ", i);
    }

    printf("\nOdd numbers in standard order: ");
    for (int i = start; i <= end; i++) {
        if (i % 2 != 0)
            printf("%d ", i);
    }

   
}

